### ISSUE TITLE
Gradient-based Hyperparameter Optimization through Reversible Learning

### ISSUE THREAD

**Issue by makdoudN** [makdoudN/hypersacred#5]
Could be fun but impose a opinionated choice of front end for deep learning (new branch for develop an opinated pytorch hypersacred based on my personal workflow)

Refs:
     https://github.com/HIPS/hypergrad

**Comment by makdoudN** [comment]
Could be fun bot not sure if this is valuable for this repository
